### DISCUSSION TITLE
Need to know about SignalR websocket and http end points

### DISCUSSION THREAD

**Discussion by mmubashirkhan1275** [dotnet/aspnetcore#27092] [Q&A] [answered]
we are using @microsoft/signalr as js client to an asp.net core api service hosting signalr hub that has ocelot gateway in between. we need to whitelist signalr endpoints on our gateway so that nothing else passes through it other than signalr. can someone please let me know about signalr endpoints considering our MapHub as /serverHub defined as the end point. we need to know all fallback and conventional websocket api calls that can occur via js client so that we can whitelist.

**Comment by BrennanConroy** [comment, marked as answer]
The only endpoints used by SignalR are `/serverHub` and `/serverHub/negotiate`.

**Reply by mmubashirkhan1275** [reply]
Thank You
